### PR TITLE
wip(): Working example in Rev cluster, changes needed

### PR DESCRIPTION
**Pull request by aacecandev**
## Description

Changes needed to make kdl-project-template work in the Rev EKS cluster

## Motivation and Context

We need to make some adjustments in the template. It's outdated and it failed in a live demo for Rev

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other changes (ci configuration, documentation or any other kind of changes)

## Checklist:

- [x] I have created tests for my code changes, and the tests are passing.
- [x] I have executed the pre-commit hooks locally.
- [ ] My change requires a change to the documentation (create a new issue if the documentation has not been updated).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: sonar-project.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-sonar.projectKey=konstellation-io_kdl-project-template
+sonar.projectKey=konstellation-io_alejandro-test
```
Comment (blopezpi) [inline sonar-project.properties:1]: Is this change needed?

Comment (aacecandev) [reply sonar-project.properties:1]: The change is not needed, but we have to be aware of every single point where the template has hardcoded values. I think the documentation would be the best place to indicate to users of the template where specific changes need to be made, but I prefer to generate a conversation about this.

**Comment by aacecandev** [comment]
closing as changes are already done in #48
